client/app/components/OnboardingTopBar/OnboardingTopBar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+
+import css from './OnboardingTopBar.scss';
+
+class OnboardingTopBar extends React.Component {
+
+  t(translationKey) {
+    return this.props.translations[translationKey];
+  }
```

Comment:
@kpuputti @mporkola 
I have that curried version on my branch - and it might make sense to move it to some utils-file. However, HoC might make more sense, but since these translations still need more thinking I'd suggest that we don't spend time to building it yet. 
(Thinking part: how to add links and other interpolation stuff)